src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -267,6 +267,12 @@
     <PackageReference Include="Microsoft.CodeAnalysis.RulesetToEditorconfigConverter" Version="3.0.0" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Microsoft.VisualBasic">
+      <HintPath>..\..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\Microsoft.VisualBasic.dll</HintPath>

```

Comment:
What is this used for?

Reply:
Nice catch. Test code was pushed without removal.
Removed.